snpdb/signals/scv_search.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
+
+from classification.models import ClinVarExport
+from library.enums.log_level import LogLevel
+from snpdb.search import search_receiver, SearchInputInstance, SearchExample, HAS_SCV, SearchMessageOverall
+
+
+@search_receiver(
+    search_type=ClinVarExport,
+    pattern=HAS_SCV,
+    example=SearchExample(
+        note="SCVnumber",
```

Comment:
Can the note be "SCV (Submitted ClinVar Record)" and the example be a little more realistic looking with a few more 0s at the front of it "SCV000056789"